runtime/vm/KeyHashTable.c

```diff
@@ -333,7 +342,7 @@ hashClassTableNew(J9JavaVM *javaVM, U_32 initialSize)
 }
 
 J9Class *
-hashClassTableAt(J9ClassLoader *classLoader, U_8 *className, UDATA classNameLength)
+hashClassTableAt(J9ClassLoader *classLoader, U_8 *className, UDATA classNameLength, BOOLEAN ignoreHiddenClass)
```

Comment:
I should have mentioned this earlier, but you can make: 

```
hashClassTableAtImpl(J9ClassLoader *classLoader, U_8 *className, UDATA classNameLength, BOOLEAN ignoreHiddenClass) { }
```

and then

```
hashClassTableAt(J9ClassLoader *classLoader, U_8 *className, UDATA classNameLength) 
{
    return hashClassTableAtImpl(classLoader, className, classNameLength, TRUE); 
}
```

So you dont need to update all the places its used.

Reply:
Updated my PR. I added `hashClassTableAtImpl` to `vm_api.h`, is that what I'm supposed to do?